content/en/docs/instrumentation/erlang/getting-started/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
@svrnm - could you rename this file to `content/en/docs/instrumentation/erlang/getting-started.md`? There seems to be no reason for it to be a section.

Reply:
good call, let me do that!

Reply:
done, also renamed instrumentation.md to manual.md

Reply:
+1